include/executors/executor_sycl.hpp

```diff
@@ -263,7 +262,7 @@ class Executor<SYCL> {
    */
   Executor(cl::sycl::queue q) : q_interface(q){};
 
-  cl::sycl::queue sycl_queue() const { return q_interface.sycl_queue(); }
+  inline Queue_Interface<SYCL> &policy_handler() { return q_interface; }
```

Comment:
this should not return a reference, it will be incorrect (reference to private member)

Reply:
fixed